release-notes/6.0/install-windows.md

```diff
@@ -0,0 +1,34 @@
+# .NET 6 installation instructions for Windows
+
+The following instructions demonstrate installing .NET 6 on Windows. These instructions augment the more general [.NET install instructions](install.md), including installing with `.zip` files, that work on multiple operating systems.
+
+## Distributions
+
+The following distributions are available for Windows:
+
+- .NET SDK: includes tools for building and testing applications, and includes the runtime distributions that follow.
+- .NET Runtime: includes the .NET runtime and libraries, enabling running console applications.
+- .NET Desktop Runtime: includes the .NET runtime and Windows desktop libraries, enabling running console, Windows Forms, and Windows Presentation Framework (WPF) applications.
+- ASP.NET Core Runtime: includes the .NET and ASP.NET Core runtimes, enabling running console, and web applications.
+- ASP.NET Core Hosting Bundle: includes the ASP.NET Core runtime and IIS support (for running both in- and out-of-process with IIS).
+
+You are recommended to install the .NET SDK to develop and build applications, and to install one of the runtimes packages (like ASP.NET Core) to exclusively run applications.
+
+## Downloads
+
+Each of the distributions can be downloaded from:
+
+- [Microsoft .NET website](https://dotnet.microsoft.com/download/dotnet/6.0)
+- [.NET 6 release notes](README.md)
+
+[Container images](https://hub.docker.com/r/microsoft/dotnet/) are provided for Windows (Nano Server, and Server Core) and Linux (Alpine, Debian, and Ubuntu).
```

Comment:
```suggestion
[Container images](https://hub.docker.com/r/microsoft/dotnet/) are provided for Windows (Nano Server and Server Core) and Linux (Alpine, Debian, and Ubuntu).
```

Reply:
Is the mention of `and Linux (Alpine, Debian, and Ubuntu)` intentional?  Is it for the scenario of running Linux containers on a Windows client machine?

Reply:
This is where host and guest get confused. I'm talking about guests.